README.md

```diff
@@ -88,7 +88,7 @@ is to feel natural for somebody used to coding in Python, and reasonably easy to
 who is not a pythonista.
 
 The dependencies of the operator framework are kept as minimal as possible; currently that's Python
-3.8 or greater, and `PyYAML` (both are included by default in Ubuntu's cloud images from 20.04 on).
+3.8 or greater, and a couple of python libraries outlined in [requirements.txt](requirements.txt).
```

Comment:
Nit: python -> Python. Maybe "and the small number of Python libraries referenced in requirements.txt"